models/staging/hubspot_staging/stg_rag_hubspot__engagement_deal.sql

```diff
@@ -38,6 +38,7 @@ final as (
     select
         engagement_id,
         deal_id,
+        engagement_type,
```

Comment:
Don't forget to add this to the staging yml

Reply:
Added!